examples/pybind11/use_dpctl_sycl_kernel/example.py

```diff
@@ -0,0 +1,41 @@
+#                      Data Parallel Control (dpctl)
+#
+# Copyright 2020-2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# coding: utf-8
+
+import use_kernel as eg
+
+import dpctl
+import dpctl.program as dppr
+import dpctl.tensor as dpt
+
+q = dpctl.SyclQueue()
+
```

Comment:
Some documentation is warranted. I know what the example is doing, but not every user will know or should be expected to know just by reading the code.